.github/workflows/buildRelease.yml

```diff
@@ -86,6 +86,76 @@ jobs:
         name: wheels
         path: dist
 
+  build_linux_arm64:
```

Comment:
In a separate PR, is it worthwhile to put the parts that are common across all hosts into a separate, parameterized action?  It is getting harder to know what changed between the build_linux and build_linux_arm64 jobs.

Reply:
Yeah this may be a nice cleanup